src/services/user.service.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-return-await */
+
+const { User } = require('../models/User.model');
+
+const getAll = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getById = async (id) => {
+  return await User.findByPk(id);
+};
+
+const createUser = async (name) => {
+  return User.create({ name });
+};
+
+const remove = async (id) => {
+  await User.destroy({
+    where: {
+      id,
+    },
+  });
+};
+
+const update = async ({ id, name }) => {
+  await User.update({ name }, { where: { id } });
```

Comment:
Ensure that the `name` is validated before updating the user. Consider adding validation logic to check if `name` is a non-empty string before proceeding with the update.